WriteVHDLSyntax.py

```diff
@@ -968,7 +1032,9 @@ def writeTBControlSignals(memDict, memInfoDict, initial_proc, final_proc, notfin
                 if memMod.is_binned:
                     if "VMSME_D" in mem:
                         disk = "DISK"
-
+                if "AS" in mem and "n1" in mem and split:
+                    string_ctrl_signals += "  signal "+mem+"_stream_V_dout : std_logic_vector(36 downto 0) := (others => '0');\n"
```

Comment:
@jasonfan393 Can we avoid the hard-coded `36`?
```suggestion
                    string_ctrl_signals += "  signal "+mem+"_stream_V_dout : std_logic_vector(" + str(memMod.bxbitwidth + memMod.bitwidth - 1) + " downto 0) := (others => '0');\n"
```

Or maybe the type should be defined in memUtil_pkg.vhd?